assets/css/dashboard/new-pa-message/select-stations-page.scss

```diff
@@ -0,0 +1,199 @@
+// Because this page is not a unique route, we cannot conditionally render
+// Sidebar the same way we do elsewhere.
+.sidebar-container:has(
+    + .page-content > .new-pa-message > .select-stations-page
+  ) {
+  display: none !important;
```

Comment:
This was a super interesting idea so I ran with it. Page looks the same as before but has all the functionality of a modal. The only thing I'm curious about a user accidentally pressing `Esc` and losing their progress. Luckily that piece is easy to take out (just remove the `onHide` prop).

Reply:
Nice. Maybe if there's a concern about losing changes there could be a confirmation ("Really discard changes to selected zones?"). Though if we didn't already have a confirmation in the design for the Cancel button itself, I don't think pressing `Esc` poses more of a risk (if anything, it's harder to do that accidentally than it is for your mouse to slip and hit Cancel instead of Review Zones).